docs/content/users/extend/additional-services.md

```diff
@@ -134,7 +150,7 @@ Anyone can create an add-on for `ddev add-on`. See [this screencast](https://www
 The `install.yaml` is a simple YAML file with a few main sections:
 
 * `name`: The add-on name. This will be used in the various `ddev add-on` commands. If the repository name is `ddev-solr`, for example, a good `name` will be `solr`.
-* `pre_install_actions`: an array of Bash statements or scripts to be executed before `project_files` are installed. The actions are executed in the context of the target project’s root directory.
+* `pre_install_actions`: an array of Bash statements or scripts to be executed before `project_files` are installed. The actions are executed in the context of the target project’s root directory. With DDEV v1.23.5+, variables from the `.ddev/.env.name` file, where `name` is the name of the add-on, are available in this context.
```

Comment:
```suggestion
* `pre_install_actions`: an array of Bash statements or scripts to be executed before `project_files` are installed. The actions are executed in the context of the target project’s root directory. With DDEV v1.23.5+, variables from the `.ddev/.env.name` file, where `name` is the name of the service in the add-on's `docker-compose.*.yaml`, are available in this context.
```

Reply:
And here `.env.name` is always the name of the add-on. I can't use the service name here because add-ons may not have services, and I can't get the service name without running `docker-compose config`, which creates a catch-22.